core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,33 @@
+const acceptanceTests = require("./utility-functions/puppeteer_utils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+
+const blogEditOptions = "e2e-test-blog-post-edit-box";
+
+async function blogDrafts_journey() {
+  const obj = await new acceptanceTests();
+  const page = await obj.init();
+
+  await page.goto(testConstants.URLs.home);
+  await obj.clickOn("button", "OK");
+  await obj.clickOn("span", "Sign in");
+  await obj.type(testConstants.SignInDetails.inputField, "testadmin@example.com");
+  await obj.clickOn("span", "Sign In");
+  
+  await page.waitForSelector(testConstants.Dashboard.MainDashboard);
```

Comment:
As a high-level user, I don't really know what this `waitForSelector` means -- that's a very technical way of putting it and I don't think it maps clearly to an end user thing.

Also I'm a bit confused about when to use `page` and when to use `obj`. This relates to the conceptual issues I mentioned above.

Reply:
done.. obj is my actor that is testing the functionalities by doing the required activities, so renamed it to user